CONTRIBUTING.md

```diff
@@ -1,23 +1,33 @@
 # Contributing to Athens
 
+If you're looking for somewhere to start contributing, check out the issues tagged with "first issue" found in [issues](https://github.com/athensresearch/athens/issues). If you have a specific feature that you would like to work on (for which there is no open issue yet), please let us know.
+
 These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
 
-Please first read the [Code of Conduct](https://github.com/tangjeff0/athens/blob/master/CODE_OF_CONDUCT.md).
+Before contributing, please first read the [Code of Conduct](https://github.com/tangjeff0/athens/blob/master/CODE_OF_CONDUCT.md).
```

Comment:
Update link to `https://github.com/athensresearch/athens/blob/master/CODE_OF_CONDUCT.md`.

Reply:
Nice catch 😃